docs-source/content/quickstart/install.md

```diff
@@ -24,9 +24,33 @@ subjects:
 EOF
 ```
 
+#### Use `Helm` to install the Operator and [Traefik](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/charts/traefik/README.md) load balancer. Use the [values.yaml](http://github.com/oracle/weblogic-kubernetes-operator/blob/master/kubernetes/samples/charts/traefik/values.yaml) in the sample but set `kubernetes.namespaces` specifically.
```

Comment:
Looks odd to make this whole sentence a title.  

To see how your edits look, you can start a hugo web server on top of your source tree.   See docs-source/runlocal.sh or runpublic.sh